src/components/key_pad_menu/_key_pad_menu.scss

```
@@ -32,14 +32,31 @@
     border-color $euiAnimSpeedFast ease-in,
     box-shadow $euiAnimSpeedFast ease-in;
 
-  &:hover,
-  &:focus {
-    @include euiSlightShadow;
-    border-color: $euiBorderColor;
+  &:not(:disabled) {
+    &:hover,
+    &:focus {
+      @include euiSlightShadow;
+      border-color: $euiBorderColor;
 
-    .euiKeyPadMenuItem__icon {
-      transform: translateY(0);
+      .euiKeyPadMenuItem__icon { // sass-lint:disable-line nesting-depth
+        transform: translateY(0);
+      }
     }
+
+  }
+
+  &:disabled {
+    color: $euiButtonColorDisabled;
+
+    &:hover,
+    &:focus {
+      cursor: not-allowed;
+    }
+
+    .euiIcon path {
```

Just caught this, but you shouldn't target `path` here because it could be any SVG shape. 

```suggestion
    .euiIcon {
```

Also since anything could be passed as a child of the keypad menu item, consider using the same filter approach as well to the whole item.

---

Made that change, also realized it's better to target `.euiKeyPadMenuItem__icon` instead of `.euiIcon`.